examples/map.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey"
+)
+
+// the questions to ask
+var simpleQs = []*survey.Question{
+	{
+		Name: "name",
+		Prompt: &survey.Input{
+			Message: "What is your name?",
+		},
+		Validate: survey.Required,
+	},
+	{
+		Name: "color",
+		Prompt: &survey.Select{
+			Message: "Choose a color:",
+			Options: []string{"red", "blue", "green"},
+		},
+		Validate: survey.Required,
+	},
+}
+
+func main() {
```

Comment:
Here's what I mean... if you have this test:
```go
package survey

import (
	"io"
	"io/ioutil"
	"os"
	"testing"

	"github.com/AlecAivazis/survey/core"
	"github.com/stretchr/testify/assert"
)

func init() {
	// disable color output for all prompts to simplify testing
	core.DisableColor = true
}

func TestStructInt(t *testing.T) {
	q := []*Question{
		{
			Name: "age",
			Prompt: &Input{
				Message: "What is your age?",
			},
			Validate: Required,
		},
	}

	in, _ := ioutil.TempFile("", "")
	defer in.Close()

	os.Stdin = in

	io.WriteString(in, "21\n")
	in.Seek(0, os.SEEK_SET)

	ans := struct{ Age int64 }{}

	err := Ask(q, &ans)

	assert.Nil(t, err)
	assert.Equal(t, int64(21), ans.Age)

}
```
this is what you get when you run it:
```
=== RUN   TestStructInt
[0G? What is your age? 21
[1F[0G? What is your age? 21
--- FAIL: TestStructInt (0.00s)
	assertions.go:241: 
                        
	Error Trace:	struct_int_test.go:41
		
	Error:		Expected nil, but got: &errors.errorString{s:"reflect.Set: value of type string is not assignable to type int64"}
		
	assertions.go:241: 
                        
	Error Trace:	struct_int_test.go:42
		
	Error:		Not equal: 21 (expected)
			        != 0 (actual)
		
FAIL
exit status 1
FAIL	command-line-arguments	0.003s
```